python/Sentinel1_Stats/Sentinel1_Stats.ipynb

```diff
@@ -0,0 +1,2372 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "849c4388-8820-4a11-9588-71c797a0a9f1",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "                <script type=\"application/javascript\" id=\"jupyter_black\">\n",
+       "                (function() {\n",
+       "                    if (window.IPython === undefined) {\n",
+       "                        return\n",
+       "                    }\n",
+       "                    var msg = \"WARNING: it looks like you might have loaded \" +\n",
+       "                        \"jupyter_black in a non-lab notebook with \" +\n",
+       "                        \"`is_lab=True`. Please double check, and if \" +\n",
+       "                        \"loading with `%load_ext` please review the README!\"\n",
+       "                    console.log(msg)\n",
+       "                    alert(msg)\n",
+       "                })()\n",
+       "                </script>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# !pip install jupyter-black jupyter\n",
+    "\n",
+    "import jupyter_black\n",
+    "%load_ext jupyter_black\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0021d684-bcc4-4e68-af6a-2655994d63c1",
+   "metadata": {},
+   "source": [
+    "# Publishing an openEO process as a service \n",
+    "\n",
+    "In this notebook, we want to show how the openEO algorithm can be shared among users. Here, we make use of an apply_dimension process, avoiding the need for extensive merge_cubes. \n",
```

Comment:
The main goal is how to create a UDP. 
Sharing among users, which is not a standard feature technically, is at most a secondary goal


Also the sentence that connects "apply_dimension" to "avoiding  extensive merge_cubes" is far from obvious and needs a lot more explanation